book/src/help_bn.md

```diff
@@ -585,6 +585,10 @@ Flags:
           server on localhost:5052 and import deposit logs from the execution
           node. This is equivalent to `--http` on merge-ready networks, or
           `--http --eth1` pre-merge
+      --subscribe-all-data-column-subnets
```

Comment:
If we are going to commit to this flag, is this the best name? Another angle is to have a `--supernode` flag. I don't have a strong opinion but now is the time to think it twice

Reply:
I like `--supernode` better

Reply:
Just dropping some thoughts here - I thought about a few alternatives when originally picking this name
- `--custody-subnet-count 12`: number of column subnets to custody, this is flexible but we don't really have a use case, and isn't really compatible with the upcoming validator custody and custody group changes.
- `--subscribe-all-data-column-subnets`: this name is consistent with the attestation subnet flag, but thinking about this now, it's actually a bit different, because it means subscribe all subnets + import all data columns + reconstruction + distributed blob building etc. so `--supernode` does feel a bit more suitable.
- `--min-data-column-custody <number|all>`: with the upcoming validator custody change, the number of custody columns may change during runtime, depending on the number of validators registered. Note I dropped the `subnet` here because of the upcoming change to decouple subnets from custody.

We've used the `supernode` term a lot between implementers, but feels a bit vague for people without the PeerDAS context, e.g. would a regular user think it subscribes to all attestation subnets and import all attestations too, or would it have a higher `--network-load` value? 

The downside is that it is a bit rigid and we may need to add more flags, if one wants to run a lower bandwidth and lower storage supernode, they _may_ want to subscribe to just 50% of the subnets - just to be able to locally perform reconstruction without having to query peers when having to serve `blob_sidecars` over HTTP.

Btw all other clients have adopted the same thing so far:

```yaml
participants:
# Supernodes
  - cl_type: prysm
    cl_image: ethpandaops/prysm-beacon-chain:peerdas-initial-sync
    cl_extra_params: [--subscribe-all-subnets]
  - cl_type: lighthouse
    cl_image: ethpandaops/lighthouse:das-devnet-2
    cl_extra_params: [--subscribe-all-data-column-subnets]
  - cl_type: teku
    cl_image: ethpandaops/teku:nashatyrev-das
    cl_extra_params: [--p2p-subscribe-all-custody-subnets-enabled]
  - cl_type: nimbus
    cl_image: ethpandaops/nimbus-eth2:peerdas-p2p
    cl_extra_params: [--subscribe-all-subnets=true]
```

Reply:
we should probably hide all peerdas flags at this stage?

Reply:
btw I also don’t have a strong opinion, happy to switch to supernode, and I think we can still change our mind before go live

Reply:
Updated help and made this a hidden flag.